src/test/skript/junit/EvtUnknownCommand.sk

```diff
@@ -0,0 +1,2 @@
+on unknown command:
+	set unknown command message to "Hey, this command does not exist."
```

Comment:
this test won't ever get called on its own, you need to call the event in the java JUnit package and then ensure that you can modify the unknown command message correctly. look at other expressions for examples

Reply:
Does this fire for console commands? You might be able to get away with `execute console command "fakecommand"` in a test